src/api/libcellml/analyserequationast.h

```diff
@@ -38,177 +38,104 @@ class LIBCELLML_EXPORT AnalyserEquationAst
-     *    - FLOOR: the floor function;
-     *    - MIN: the minimum function;
-     *    - MAX: the maximum function;
-     *    - REM: the remainder function;
-     *  - Calculus elements:
-     *    - DIFF: the differentiation operator;
-     *  - Trigonometric operators:
-     *    - SIN: the sine function;
-     *    - COS: the cosine function;
-     *    - TAN: the tangent function;
-     *    - SEC: the secant function;
-     *    - CSC: the cosecant function;
-     *    - COT: the cotangent function;
-     *    - SINH: the hyperbolic sine function;
-     *    - COSH: the hyperbolic cosine function;
-     *    - TANH: the hyperbolic tangent function;
-     *    - SECH: the hyperbolic secant function;
-     *    - CSCH: the hyperbolic cosecant function;
-     *    - COTH: the hyperbolic cotangent function;
-     *    - ASIN: the arc sine function;
-     *    - ACOS: the arc cosine function;
-     *    - ATAN: the arc tangent function;
-     *    - ASEC: the arc secant function;
-     *    - ACSC: the arc cosecant function;
-     *    - ACOT: the arc cotangent function;
-     *    - ASINH: the arc hyperbolic sine function;
-     *    - ACOSH: the arc hyperbolic cosine function;
-     *    - ATANH: the arc hyperbolic tangent function;
-     *    - ASECH: the arc hyperbolic secant function;
-     *    - ACSCH: the arc hyperbolic cosecant function;
-     *    - ACOTH: the arc hyperbolic cotangent function;
-     *  - Piecewise statement:
-     *    - PIECEWISE: the "piecewise" statement;
-     *    - PIECE: the "piece" part of a "piecewise" statement;
-     *    - OTHERWISE: the "otherwise" part of a "piecewise" statement;
-     *  - Token elements:
-     *    - CI: an identifier (i.e. the name of a model variable);
-     *    - CN: a number;
-     *  - Qualifier elements:
-     *    - DEGREE: the degree of a root operator (it is only used when its
-     *      value is not 2);
-     *    - LOGBASE: the base with respect to which the logarithm is taken;
-     *    - BVAR: the bound variable of a differential equation;
-     *  - Constants:
-     *    - TRUE: the "true" boolean;
-     *    - FALSE: the "false" boolean;
-     *    - E: Euler's number;
-     *    - PI: the π constant;
-     *    - INF: the infinity value; or
-     *    - NAN: the not-a-number value.
+     * The type of a node in an abstract syntax tree (AST), i.e. whether it is
+     * an assignment type, a relational or a logical operator, an arithmetic
+     * operator, a calculus element, a trigonometric operator, a piecewise
+     * statement, a token element, a qualifier element, or a constant.
      */
     enum class Type
     {
         // Assignment.
 
-        ASSIGNMENT,
+        ASSIGNMENT, /**< The assignment operator. */
 
         // Relational and logical operators.
 
-        EQ,
-        NEQ,
-        LT,
-        LEQ,
-        GT,
-        GEQ,
-        AND,
-        OR,
-        XOR,
-        NOT,
+        EQ, /**< The equal to function. */
```

Comment:
these are operators, not functions...right? (but I can see this is just improving the structure of the existing docs, so its not a show stopper)

Reply:
Oops. Fixed in #103abc16.